src/main/java/step2/LotteryNumbers.java

```diff
@@ -0,0 +1,33 @@
+package step2;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LotteryNumbers {
+    private List<Integer> lottoNumberList;
```

Comment:
인스턴스 변수명은 직접적인 구현체가 드러나지 않도록 작성하는 편이 낫습니다:)
```suggestion
    private List<Integer> lottoNumbers;
```

Reply:
확인했습니다!